lib/stale.js

```diff
@@ -198,8 +198,27 @@ module.exports = class Stale {
   }
 
   since (days) {
-    const ttl = days * 24 * 60 * 60 * 1000
-    let date = new Date(new Date() - ttl)
+    const daysInMilliseconds = 24 * 60 * 60 * 1000
+    const now = Date.now()
+    let daysToCount = 0
+    let daysAgo = 0
+
+    const { skipWeekends } = this.config
+    if (skipWeekends) {
+      while (daysToCount < days) {
+        daysAgo++
+        const dayOfWeek = new Date(now - daysAgo * daysInMilliseconds).getDay()
+        if (dayOfWeek !== 0 && dayOfWeek !== 6) {
+          daysToCount++
```

Comment:
This is a very minor nit, but I feel like there's a more performant algorithm for calculating this besides iterating over every day. 

I'm not really smart enough to figure it out without messing with it for a while, but I'd try something like this:

- `Math.floor(days / 7) * 2` = minimum number of weekend days
- `number of days % 7` = number of additional days
- some maths taking to figure out how many of the additional days are weekends based on the current day of the week